feat(PricingSlot): custom accessors

Coriou opened the pull request:
PricingSlot has two new props: titleAccessor & priceAccessor, allowing to pass custom price & title components respectively

Not ready to be merged, but is working. Check the example:

```bash
$ npm start
# localhost:8080
```

To do:

- [x] Write / update tests
- [x] Document readme
- [x] Provide an example

Coriou committed:
feat(PricingSlot): custom accessors

PricingSlot has two new props: titleAccessor & priceAccessor, allowing to pass custom price & title components respectively

Coriou commented:
Spamming with PRs here. For a quick check of everything in #11, #10 & #9, you can have a look at [coriou/react-pricing-table#develop](https://github.com/Coriou/react-pricing-table/tree/develop)

gergely-nagy commented:
@Coriou thanks a ton for this pr and all the updates.

I will get to reviewing them today.

Coriou commented:
Hey, have you had some time to take a look yet ? Thanks

Coriou commented:
That was fast :) 

I'll make myself available in Gitter if you need to discuss this or help with merging all the PRs

Coriou committed:
feat(PricingSlot): custom accessors - tests

Basic Jest test for custom accessors features: "should render without crashing"

Coriou committed:
feat(PricingSlot): custom accessors - readme

Update readme to match new props for PricingSlot

Coriou committed:
feat(PricingSlot): custom accessors - readme fix

Oops, it's an HTML table - need to use <code /> instead of backticks

gergely-nagy commented:
Thanks for your work! Trying to merge this PR ASAP 😄
